mesonbuild/dependencies/ui.py

```diff
@@ -242,7 +241,7 @@ def __init__(self, name: str, environment: 'Environment', kwargs: T.Dict[str, T.
                                                                  low=0, high=None, guess=e,
                                                                  prefix='#include <vulkan/vulkan.h>',
                                                                  env=environment,
-                                                                 extra_args=None,
+                                                                 extra_args=extra_args,
```

Comment:
This could just be `extra_args=self.compile_args` to avoid the extra assignment.